docs/src/tutorials/exposing_new_intrinsics.jl

```diff
@@ -0,0 +1,101 @@
+# # Introduction
```

Comment:
This is too detailed for a tutorial. Maybe add it to a new top-level `Hacking` section below `Development`?